packages/editor/src/components/Editor.tsx

```diff
@@ -219,6 +219,7 @@ export const Editor: React.FC<Props> = observer(
                 display="flex"
                 flexDirection="column"
                 textAlign="left"
+                lazyBehavior='keepMounted'
```

Comment:
Can we make this as property? I think it is something like `visually` of hidden trait. Sometimes we don't want other tab components mount.

Reply:
changed